DocumentDataAPI/DocumentDataAPI/Data/Mappers/EntityMapExtensions.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Linq.Expressions;
+using System.Reflection;
+using Dapper.FluentMap.Mapping;
+
+namespace DocumentDataAPI.Data.Mappers;
+
+public static class EntityMapExtensions
+{
+    public static string GetMappedColumnName<T>(this EntityMap<T> entityMap, string propertyName) where T : class
```

Comment:
Det her er en [extension method](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/extension-methods) for de interesserede